backend/controllers/authController.js

```diff
@@ -47,6 +47,20 @@ exports.signup = catchAsync(async (req, res, next) => {
     passwordConfirm: req.body.passwordConfirm,
   });
 
+  try {
+    await sendEmail({
+      emailAddress: newUser.emailAddress,
+      subject: "Welcome to Average JS Webshop",
+      message:
+        "Welcome to Average JS Webshop! We are thrilled to have you as a new member of our online community. Thank you for choosing us for your shopping needs.",
+    });
+  } catch (err) {
+    return next(
+      new AppError("There was an error sending the email. Try again later!"),
```

Comment:
Ezt az error message-t a frontend (React) milyen úton éri el?

Reply:
Pl. így:

`axios.post('/api/signup', userData)
    .then(response => {
        // Handle successful response
    })
    .catch(error => {
        if (error.response) {
            // The request was made and the server responded with a status code
            // that falls out of the range of 2xx
            const errorMessage = error.response.data.message;
            console.log(errorMessage); // Or display it using React state, etc.
        }
        // Handle other types of errors if needed...
    });
`